lib/ansible/galaxy/collection/concrete_artifact_manager.py

```diff
@@ -510,7 +501,7 @@ def _get_meta_from_installed_dir(
         with open(b_manifest_json_path, 'rb') as manifest_fd:
             b_manifest_txt = manifest_fd.read()
     except OSError:
-        raise LookupError(
+        raise AnsibleError(
```

Comment:
This shouldn't raise a base error, when I added it, I've set up the translations on the calling side.